Apps/Sandcastle/gallery/development/3D Models Outline.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Create 3D models using glTF.">
+    <meta name="cesium-sandcastle-labels" content="Development">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #toolbar {
+        background: rgba(42, 42, 42, 0.8);
+        padding: 4px;
+        border-radius: 4px;
+    }
+    #toolbar input {
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+    <table>
+    <tbody>
+    <tr>
+        <td>Size</td>
+        <td>
+            <input type="range" min="0" max="20" step="1" data-bind="value: highlightSize, valueUpdate: 'input'">
+            <input type="text" size="5" data-bind="value: highlightSize">
+        </td>
+    </tr>
+    <tr>
+        <td>Opacity</td>
+        <td>
+            <input type="range" min="0" max="100" step="1" data-bind="value: highlightOpacity, valueUpdate: 'input'">
+            <input type="text" size="5" data-bind="value: highlightOpacity">
+        </td>
+    </tr>
+
+</tbody></table>
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    infoBox : false,
+    selectionIndicator : false,
+    shadows : true,
+    // Explicitly request a stencil buffer so highlighting will work.
+    contextOptions: {
+        webgl : {
+            stencil: true
```

Comment:
@bagnell is this needed since polygons on terrain also require stencil?  Last time I reviewed this, it was not clear to me where Cesium requested stencil.  Seems like it should default to `true` now.